frontend/src/app/general/auth/auth/auth.component.ts

```diff
@@ -22,12 +22,12 @@ export class AuthComponent implements OnInit {
     }
   }
 
-  authProvider = environment.authentication;
   public params = {} as ParamMap;
 
   constructor(
     private authService: AuthService,
     private pageLayoutService: PageLayoutService,
+    public metadataService: MetadataService,
```

Comment:
Thats just a personal opinion but I prefer to have the services sorted by their visibility in the constructor (i.e., all public services at the top followed by the private ones). This helps directly identify and distinguish public and private services. What do you think about this?

Reply:
Done, is changed.